lib/sprangular.rb

```diff
@@ -7,14 +7,11 @@
 require 'rabl'
 require 'jquery-rails'
 require 'angularjs-rails'
-require 'rails-assets-bootstrap-sass-official'
 require 'rails-assets-underscore'
 require 'rails-assets-underscore.string'
-require 'rails-assets-angular-bootstrap'
-require 'rails-assets-angular-strap'
-require 'rails-assets-angular-motion'
-require 'rails-assets-bootstrap-additions'
 require 'rails-assets-angularytics'
+require 'bourbon'
+require 'neat'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.